bsp/boards/i2c.h

```diff
@@ -33,10 +33,10 @@ void i2c_write_register(uint8_t bus_num, uint8_t slave_addr, uint8_t length, uin
 uint8_t i2c_slave_present(uint8_t bus_num,uint8_t slave_address);
 
 void i2c_read_byte(uint8_t address, uint8_t* byte);
-void i2c_read_bytes(uint8_t address, uint8_t* buffer, uint32_t length);

```

Comment:
I guess nothing special, it was just convenient to do it since the signature of the function seemed more straight forward to implement. I guess we can change that to use the standard ones, as you prefer.